GetandTake/Program.cs

```diff
@@ -1,10 +1,24 @@
+using AutoMapper;
+using GetandTake.DataAccess.Repositories;
+using GetandTake.DataAccessLayer.EF;
+using GetandTake.Services.Abstracts;
+using GetandTake.Services.AutoMapper;
+using GetandTake.Services.Concretes;
+using Microsoft.EntityFrameworkCore;
+using System;
+
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddDbContext<NorthwindDbContext>(
+    options =>  options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
 // Add services to the container.
 builder.Services.AddRazorPages();
+builder.Services.AddTransient<IProductService, ProductManager>();
```

Comment:
It's better to mark services with `AddScoped` lifecycle since services might be reused in controllers several times and there is no sense to create their object  twice.